src/OrchardCore.Modules/OrchardCore.Twitter/Signin/Configuration/TwitterOptionsConfiguration.cs

```diff
@@ -78,10 +78,7 @@ public void Configure(string name, TwitterOptions options)
             }
             catch
             {
-                if (_shellSettings.State == TenantState.Running)
-                {
-                    _logger.LogError("The Consumer Secret could not be decrypted. It may have been encrypted using a different key.");
-                }
+                _logger.LogError("The Twitter Consumer Secret could not be decrypted. It may have been encrypted using a different key.");
```

Comment:
You also removed the shell state check. Is it really not necessary, just here?

Reply:
Yes i added it at some point, then reverted it to make it consistent with all others configuration files.

This is in the `GetSettingsAsync()` that we now check the shell state if the settings are not validated to prevent a warning, see line 102 for this file, as done in other option configuration files. And in that case we return a null value that is checked in this file line 69, i added this check as it was already done in most of the other configuration files.

Reply:
Ah I see, of course, OK then.